components/script/dom/bindings/trace.rs

```diff
@@ -860,6 +860,12 @@ unsafe impl JSTraceable for StyleLocked<MediaList> {
     }
 }
 
+unsafe impl JSTraceable for JSScript {
```

Comment:
I think we're going to have to store the script in a `Heap`, and then this won't be necessary, see https://github.com/servo/rust-mozjs/blob/11cabdef2cbf0884a2cc33e3c73719646bd31ce5/src/rust.rs#L465